public/index.html

```diff
@@ -9,9 +9,19 @@
   <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto">
   <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/1.2.0/semantic.min.js"></script>
 </head>
-
 <body>
-<div id="app" className="container"></div>
+<div id="app">
+  <p class="meetup-date"></p>
```

Comment:
Yes that can definitely be removed. It was a placeholder but I just forgot to remove it